jupyter_notebook.sh

```diff
@@ -35,16 +42,11 @@ function import_anonymous_data {
 	rm -r ${TMP_DIR}
 }
 
-function tear_down {
-	docker-compose down
-}
-
 function exec_mysql {
```

Comment:
As far as I can see, this function is not being used. Can we get rid of it?

Reply:
you mean `exec_mysql`? It is for example used in https://github.com/serlo/evaluations/blob/3b496b2ddc3f9cb62b59a19601ac1db7ffadda03/jupyter_notebook.sh#L35

Reply:
Oh, sorry my bad!